community_images/httpd/bitnami/functional_test.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -x
+set -e
+
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# shellcheck disable=SC1091
+. "${SCRIPTPATH}"/../../common/helpers.sh
+
+HELM_RELEASE=apache-release
+NAMESPACE=$(get_namespace_string "${HELM_RELEASE}")
+REPOSITORY=apache
+IMAGE_REPOSITORY=rapidfort/"$REPOSITORY"
+
+k8s_test()
+{
+    # setup namespace
+    setup_namespace "${NAMESPACE}"
+
+    # install helm
+    with_backoff helm install "${HELM_RELEASE}" bitnami/"$REPOSITORY" --set image.repository="$IMAGE_REPOSITORY" --namespace "${NAMESPACE}"
+    report_pulls "${IMAGE_REPOSITORY}"
+
+    # wait for pods
+    kubectl wait pods "${HELM_RELEASE}"-0 -n "${NAMESPACE}" --for=condition=ready --timeout=10m
+
+    # log pods
+    kubectl -n "${NAMESPACE}" get pods
+    kubectl -n "${NAMESPACE}" get svc
+
+    # delte cluster
+    helm delete "${HELM_RELEASE}" --namespace "${NAMESPACE}"
+
+    # delete pvc
+    kubectl -n "${NAMESPACE}" delete pvc --all
+
+    # clean up namespace
+    cleanup_namespace "${NAMESPACE}"
+}
+
+docker_test()
+{
+    # create network
+    docker network create -d bridge "${NAMESPACE}"
+
+    # create docker container
+    docker run --rm -d --network="${NAMESPACE}" \
+        --name "${NAMESPACE}" "$IMAGE_REPOSITORY":latest
+    report_pulls "${IMAGE_REPOSITORY}"
+
+    # sleep for few seconds
+    sleep 30
+
+    # get docker host ip
+    HTTPD_HOST=$(docker inspect "${NAMESPACE}" | jq -r ".[].NetworkSettings.Networks[\"${NAMESPACE}\"].IPAddress")
+
+    # testing using apache benchmark tool
+    ab -t 100 -n 10000 -c 10 HTTPD_HOST
```

Comment:
we may need to also change the setup.sh file to include this tool since it may not be present by default

Reply:
Added the changes in setup.sh